tests/main/security-device-cgroups-strict-enforced/task.yaml

```diff
@@ -61,20 +61,28 @@ execute: |
     rm -f /var/snap/test-strict-cgroup/common/started
     rm -f /var/snap/test-strict-cgroup/common/ready
 
+    # device cgroup settings can be updated by snap-device-helper based on udev
+    # events, in which case only the device identified by the event should be
+    # changed, while no other property of the device cgroup shall be changed
+    # (incl. other already allowed devices)
+
     # trigger a change event for /dev/full so that we observe that the device
     # cgroup settings remain stable and we can still access /dev/zero
     snap run test-strict-cgroup.sh -c 'touch /var/snap/test-strict-cgroup/common/started; until test -e /var/snap/test-strict-cgroup/common/ready; do sleep 1; done; echo ok > /dev/zero' > run.log 2>&1 &
     retry -n 5 test -e /var/snap/test-strict-cgroup/common/started
     # save the dump
     tests.device-cgroup test-strict-cgroup.sh dump | sort > dump-before-change
-    # trigger a change of /dev/full
+    # by not the device cgroup got already set up by snap-confine when the
```

Comment:
This comment reads weird, is something missing here?

Reply:
s/not/now :smile:

Reply:
Oh yes, now it makes sense :)